### PR TITLE
navbar and codelabz card cypress tests

### LINKED ISSUE #168: Write cypress tests for frontend components
Write cypress tests for frontend components

### PR DESCRIPTION
**Pull request by rijusougata13**
## Description

navbar and codelabz card cypress tests
## Related Issue

Fixed #168 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rijusougata13** [comment]
what I have done :
1. Check mini navbar is only accessable to non loggedin user and main navbar to loggedin user
2. check there is necessary components in main navbar
3. check app button working fine also login button
4. check codelabz card have all button groups .

**Review by thisuraseniya — APPROVED** [review]

**Review by AsithaIndrajith — APPROVED** [review]

File: cypress/integration/components/navbar/navbar.spec.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/// <reference types="cypress" />
```
Comment (shehand) [inline cypress/integration/components/navbar/navbar.spec.js:1]: can remove this comment right?

File: src/routes.js [hunk 2]
```diff
@@ -78,19 +89,71 @@ const Routes = () => {
...
+          <Route
+            exact
+            path={"/org/:handle"}
+            component={UserIsAllowedUserDashboard(ViewOrganization)}
+          />
+          <Route
+            exact
+            path={"/editor"}
+            component={UserIsAllowedUserDashboard(Editor)}
+          />
```
Comment (shehand) [inline src/routes.js:156]: I prefer the previous way that we used to have in here more

**Review by shehand — APPROVED** [review]

File: cypress/integration/components/navbar/navbar.spec.js (hunk 1 above)
Comment (rijusougata13) [reply cypress/integration/components/navbar/navbar.spec.js:1]: no actually that define that its a test file and opens other functionalities.

File: src/routes.js (hunk 2 above)
Comment (rijusougata13) [reply src/routes.js:156]: ok I'll change prettier format

**Pull request merged**